app/models/certificate.server.ts

```diff
@@ -41,6 +42,10 @@ export function updateCertificateById(
   });
 }
 
+export function deleteAllByUsername(username: Certificate['username']) {
```

Comment:
The function `deleteAllByUsername` name is ambiguous. I would change the id to `deleteAllCertificatesByUsername` or something more explicit.

Reply:
Sure that's a good idea